test/e2e_node/conformance_test.go

```diff
@@ -0,0 +1,116 @@
+/*
+Copyright 2015 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2e_node
+
+import (
+	"fmt"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"k8s.io/kubernetes/pkg/api"
+	"k8s.io/kubernetes/pkg/api/errors"
+	client "k8s.io/kubernetes/pkg/client/unversioned"
+)
+
+var _ = Describe("Kubelet", func() {
+	var cl *client.Client
+	var terminateCase ConformanceContainer
+	var invalidImageCase ConformanceContainer
+
+	BeforeEach(func() {
+		// Setup the apiserver client
+		cl = client.NewOrDie(&client.Config{Host: *apiServerAddress})
+		terminateCase = ConformanceContainer{
+			Container: api.Container{
+				Image:           "gcc.io/google_testcontainers/busybox",
+				Name:            "busybox",
+				Command:         []string{"echo", "'Hello World'"},
+				ImagePullPolicy: api.PullIfNotPresent,
+			},
+			//terminated soon.
+			Status: "terminated",
+			Client: cl,
+		}
+		invalidImageCase = ConformanceContainer{
+			Container: api.Container{
+				Image:           "foo.com/foo/foo",
+				Name:            "foo",
+				Command:         []string{"foo", "'Should not work'"},
+				ImagePullPolicy: api.PullIfNotPresent,
+			},
+			//waiting for image pull
+			Status: "waiting",
+			Client: cl,
+		}
+	})
+
+	Describe("container conformance test", func() {
+		Context("when running a container that terminates", func() {
+			It("it should start successfully [Conformance]", func() {
+				if err := terminateCase.Create(); err != nil && !errors.IsAlreadyExists(err) {
+					Expect(err).NotTo(HaveOccurred(), err.Error())
+					return
+				}
+
+				errs := Retry(time.Minute*1, time.Second*30, func() error {
```

Comment:
I know you copied this from the other test, but instead of Retry consider using  https://onsi.github.io/gomega/#making-asynchronous-assertions 
Something like:
>
Eventually(func() bool {
    return terminateCase.IsStarted()
}, time.Minute*1, time.Second*10).Should(BeTrue())